spec/services/case_information_service_spec.rb

```diff
@@ -17,6 +17,13 @@
     expect(cases[caseinfo.nomis_offender_id]).to be_kind_of(CaseInformation)
   end
 
+  it 'can eager load the responsibility association' do
+    create(:responsibility, nomis_offender_id: caseinfo.nomis_offender_id)
+    cases = described_class.get_case_information([caseinfo.nomis_offender_id])
+
+    expect(cases[caseinfo.nomis_offender_id].association(:responsibility).loaded?).to eq(true)
```

Comment:
This is really cool! I didn't know we could test that associations are eager loaded.